src/Workspaces/Core/Portable/CodeStyle/CodeStyleOptions.cs

```diff
@@ -184,6 +184,14 @@ public class CodeStyleOptions
                     new EditorConfigStorageLocation<CodeStyleOption<AccessibilityModifiersRequired>>("dotnet_style_require_accessibility_modifiers", s => ParseAccessibilityModifiersRequired(s)),
                     new RoamingProfileStorageLocation("TextEditor.%LANGUAGE%.Specific.RequireAccessibilityModifiers")});
 
+        internal static readonly PerLanguageOption<CodeStyleOption<bool>> PreferReadonly = new PerLanguageOption<CodeStyleOption<bool>>(
```

Comment:
i can imagine this epxanding to multiple values in the future. as such, we may want to make it an enum value instead of a bool.

Reply:
What would the values now be?